### PR TITLE
Languagepack upgrade alert

### LINKED ISSUE #5005: Contentrec errors upon upgrade to 0.16
## Summary

Learner is unable to see the contentrec page upon upgrade with `Unexpected error: 'NoneType' object has no attribute '__getitem__'` error appearing.

## Branch or installer method

0.14 installation on XP, upgraded to 0.16 (build 177). 

## Screenshots

contentrec-error

Progress report, for comparison, had listed 4 topics with several videos & exercises worked on, so there should be something in the contentrec.

### PR DESCRIPTION
**Pull request by aronasorman**
## Summary

Fixes #5005. We now show this alert on the homepage if there are language packs that need to be upgraded:

screen shot 2016-03-24 at 14 55 59

Note: is shown for everyone, even if you're not logged in. Since this can potentially prevent the said languages from functioning properly, having everyone see it will hopefully convince the admin to upgrade sooner than later.

### CODE REVIEW

**Comment by aronasorman** [comment]
@radinamatic

**Comment by radinamatic** [comment]
Sounds OK, @aronasorman!

I'm going to assume the message goes away as soon as the relevant language pack(s) have been updated? :wink:

**Comment by aronasorman** [comment]
Yup! Error goes away once all languages listed are upgraded.

**Comment by aronasorman** [comment]
There's one test failing:
```
======================================================================
FAIL: test_device_unregistered (kalite.distributed.tests.browser_tests.distributed.DeviceUnregisteredTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ubuntu/ka-lite/kalite/distributed/tests/browser_tests/distributed.py", line 78, in test_device_unregistered
    self.browser_check_django_message(message_type="warning", contains="complete the setup.")
  File "/home/ubuntu/ka-lite/kalite/testing/mixins/browser_mixins.py", line 121, in browser_check_django_message
    self.assertEqual(num_messages, len(messages), msg)
AssertionError: 1 != 2 : Make sure there are 1 message(s), type='warning'.

----------------------------------------------------------------------
```

Will fix tomorrow.

File: kalite/testing/mixins/browser_mixins.py [hunk 1]
```diff
@@ -114,6 +114,9 @@ def browser_check_django_message(self, message_type=None, contains=None, exact=N
         else:
             messages = []
 
+        if isinstance(contains, str):  # make sure we only look at messages we're looking for.
```
Comment (rtibbles) [inline kalite/testing/mixins/browser_mixins.py]: These are not the droids you're looking for!

**Comment by aronasorman** [comment]
The tests, they pass.

**Pull request merged**